portfolio/src/main/webapp/index.html

```diff
@@ -95,6 +95,13 @@ <h1>Michaela Murray's Portfolio</h1>
         <option value="Ramen-Nagi">Ramen Nagi, Palo Alto, California, US</option>
         <option value="Havanna">Havanna, Cuba</option>
     </select>
+    <form action="/markers" type="POST" id="storeMarker">
+        <label>Marker I want to save:</label>
+        <input type="number" step="0.000001" min="-90" max="90" id="lat">
```

Comment:
can you add labels for each input? it will be good for accessibility.

Reply:
will do!